include/faabric/util/exec_graph.h

```diff
@@ -16,4 +16,6 @@ void incrementCounter(faabric::Message& msg,
                       const int valueToIncrement = 1);
 
 static inline std::string const mpiMsgCountPrefix = "mpi-msgcount-torank-";
+
+static inline std::string const mpiMsgTypeCountPrefix = "mpi-msgtype-torank";
```

Comment:
I can't remember what the conclusion of our last discussion on these two points was, but to reiterate (feel free to ignore) I think these constants should (a) be `#define`s to fit with the style of the repo; (b) live in the MPI headers if possible because (i) they're MPI specific; (ii) this list will continue to grow and potentiall become quite big otherwise.

Either way, re. your point below about tracking the number of fields, I think it's still useful, but to more obviously connect them up, I'd add a constant wherever these labels are defined, e.g. `#define N_MPI_EXEC_GRAPH_DETAILS 2`, then use that in the test.